apps/wolfsshd/wolfsshd.c

```diff
@@ -273,13 +263,30 @@ static int UserAuthResult(byte result,
         WS_UserAuthData* authData, void* userAuthResultCtx);
 
 
+/* Frees up the WOLFSSH_CTX struct */
+static void CleanupCTX(WOLFSSHD_CONFIG* conf, WOLFSSH_CTX** ctx,
+        byte** banner)
+{
+    if (banner != NULL && *banner != NULL) {
+#ifndef NO_FILESYSTEM
+        freeBufferFromFile(*banner, NULL);
```

Comment:
This should be in the config for the specific user I think. To handle something like this in the future:

```
Match User sshUser
    Banner /path/to/specific_banner
```

Reply:
Opps sorry, looks like it is setup that way. Just init/free different than I expected. Please double check the XFREE call though after CleanupCTX (with *banner set to NULL here).